pkg/util/azureclient/openshiftmanagedcluster/2018-09-30-preview/client.go

```diff
@@ -73,7 +73,7 @@ type OpenShiftManagedClustersUpdateTagsFuture struct {
 // If the operation has not completed it will return an error.
 func (future *OpenShiftManagedClustersUpdateTagsFuture) Result(client OpenShiftManagedClustersClient) (osmc v20180930preview.OpenShiftManagedCluster, err error) {
 	var done bool
-	done, err = future.Done(client)
+	done, err = future.DoneWithContext(context.Background(), client)
```

Comment:
why these changes?  What do they add?

Reply:
@jim-minter `github.com/Azure/go-autorest` [deprecated `Done` in v11.2.0](https://github.com/Azure/go-autorest/blob/3492b2aff5036c67228ab3c7dba3577c871db200/CHANGELOG.md#v1120) and [removed in v12.0.0](https://github.com/Azure/go-autorest/blob/3492b2aff5036c67228ab3c7dba3577c871db200/CHANGELOG.md#v1200):

```go
future.DoneWithContext(context.Background(), client)
```
is an equvivalent of

```go
future.Done(client)
```

from the older SDK versions

Reply:
See implementation: https://github.com/Azure/go-autorest/blob/v11.2.0/autorest/azure/async.go#L89-L93

Reply:
Oh I see, it's that upstream is marking it as deprecated.  Whatever ;-)

Reply:
In fact, no, these changes don't make sense.  Pulling a context out of thin air to avoid calling a deprecated function is not better than calling a deprecated function.  We're not going to put the effort into passing contexts all around here; leave the calls to Done() as they were.

Reply:
@jim-minter `Done()` no longer exists in `go-autorest`

Reply:
got it, sorry.  Then please use context.TODO() here.